src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -60,6 +59,7 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
     <PackageReference Include="System.Data.Odbc" Version="$(SystemDataOdbcVersion21)" />
+    <Compile Include="SqlConnectionStringBuilderTest.cs" />
```

Comment:
Any reason we move `SqlConnectionStringBuilderTest` here?

Reply:
When testing against .NET Standard 2.1, this exception is thrown from one of the ConnectionStringTests:

`System.ArgumentException : Keyword not supported: 'poolblockingperiod'.`

I've changed it to only be included when not testing against .NET Standard. Any suggestions on what to do?